package.json

```diff
@@ -15,10 +15,12 @@
         "dev-deploy:contract": "rm -rf neardev/ && near dev-deploy --wasmFile target/wasm32-unknown-unknown/release/mg_nft.wasm",
         "deploy:contract": "near deploy --wasmFile target/wasm32-unknown-unknown/release/mg_nft.wasm",
         "build:lib": "rm -rf lib/ && yarn tsc",
-        "lint:lib": "eslint src test"
+        "lint:lib": "eslint src test",
+        "measure:storage-costs": "npx ts-node test/measure.ts"
```

Comment:
Would it be possible to use `yarn` instead of `npx`? So we use only one dep manager. We might need to add `ts-node` to `package.json` to be able to run it locally.

Reply:
Yes, will do so now.